src/controllers/breaches.js

```diff
@@ -18,6 +22,7 @@ async function breachesPage (req, res) {
 
   const data = {
     breachesData,
+    breachLogos: await getBreachLogos(req.app.locals.breaches),
```

Comment:
Would this potentially add delay to each render? 

Could we front-load this work into the "get all breaches" function in `/utils/hibp.js`, which occurs much less frequently?  Then, perhaps it can be derived as part of the `breachesData` object?  See other comment with more discussion.

cc @mansaj for having created `breachesData`

Reply:
Let's centralise this thread here: https://github.com/mozilla/blurts-server/pull/2865#discussion_r1116373574